src/Promitor.Agents.Scraper/Validation/MetricDefinitions/AzureMetricConfigurationValidator.cs

```diff
@@ -47,5 +60,42 @@ public IEnumerable<string> Validate(AzureMetricConfiguration azureMetricConfigur
 
             return errorMessages;
         }
+
+        private IEnumerable<string> ValidateLogAnalyticsConfiguration(LogAnalyticsConfiguration logAnalyticsConfiguration)
+        {
+            var resultString = "project result";
+            var errorMessages = new List<string>();
+
+            if (logAnalyticsConfiguration == null)
+            {
+                errorMessages.Add("Invalid Log Analytics is configured");
```

Comment:
Nit for all error messages:
```suggestion
                errorMessages.Add("Invalid Azure Log Analytics is configured");
```
This is because it's a product name.